external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java

```diff
@@ -31,16 +32,23 @@
     @ConfigProperty(name = "engine.receiver.store-events")
     boolean storeEvents;
 
+    @ConfigProperty(name = "engine.receiver.log-events", defaultValue = "false")
```

Comment:
Lets keep logging properties on logging configuration.

Reply:
How do you mean?
The intention here is to switch this on/off from the env in Kube.

Reply:
You can do that with env parameters using ``quarkus.log.category."categories".level``